k8s/base/svc.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: liftbridge-headless
```

Comment:
Can you explain the liftbridge-headless vs. liftbridge services? Sorry I'm dense. :)

Reply:
Yah, of course. A headless service provided the stable network and volume ID of each member of a StatefulSet. Each pod will always have that number assigned and each volume that's grabbed by the pods for storage will take their assigned volume with them when a pod moves to another host (if the storage can be moved, that is – not so when it's bound to a particular region in GPC/EC2/Azure whatnot). See here for an explanation: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#stable-network-id.

That way you can access each member (each pod) of the StatefulSet inside the cluster using the headless service as a DNS name – e.g. liftbridge-0.liftbridge-headless.svc:9292.

Reply:
Is that answer to your satisfaction? 😬

Reply:
Makes sense, thanks!